modules/lib/lib-content/src/test/resources/test/QueryContentHandlerTest_dsl_query.js

```diff
@@ -0,0 +1,299 @@
+var assert = require('/lib/xp/testing.js');
+var content = require('/lib/xp/content.js');
+
+var expectedJson = {
+    'total': 20,
+    'count': 3,
+    'hits': [
+        {
+            '_id': 'id1',
+            '_name': 'name1',
+            '_path': '/a/b/name1',
+            'creator': 'user:system:admin',
+            'modifier': 'user:system:admin',
+            'createdTime': '1970-01-01T00:00:00Z',
+            'modifiedTime': '1970-01-01T00:00:00Z',
+            'type': 'base:unstructured',
+            'displayName': 'My Content 1',
+            'hasChildren': false,
+            'valid': false,
+            'data': {},
+            'x': {},
+            'page': {},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8692790)
```suggestion
            "page": {},
```